modules/plugin/svg/src/test/java/org/geotools/renderer/style/SVGGraphicFactoryTest.java

```diff
@@ -32,20 +37,21 @@
  */
 public class SVGGraphicFactoryTest extends TestCase {
 
-    private SVGGraphicFactory svg;
```

Comment:
Any particular reason we switched to re-instantiating a new factory each time? I don't see anywhere we modify it, and in the case where we need to instantiate one with hints we can do a special case.
This change just seems unnecessarily verbose to me.